ssc_interface_wrapper/launch/ssc_pacmod_driver.launch

```diff
@@ -72,7 +72,6 @@
     <remap from="as/turn_signal_command" to="turn_signal_command"/>
     <remap from="as/velocity_accel" to="velocity_accel"/>
     <include file="$(find as)/launch/ssc_interface.launch">
-        <arg name="use_rear_wheel_speed" value="true"/>
```

Comment:
What is the reason to take this out?

Reply:
It no longer exists in the ssc_interface launch file

Reply:
Sounds good.